frontend/src/config/patient.schema.json

```diff
@@ -0,0 +1,145 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "patient.schema.json",
+    "title": "Patient",
+    "description": "A patient record for COVID testing",
+    "type": "object",
+    "properties": {
+      "patientID": {
+        "description": "Unique patient identifier",
```

Comment:
this is the internal to PRIME identifier, yes?

Reply:
if so, i think we should also from the beginning include an array property for external identifiers, or an array for all identifiers including the PRIME app ID. In FHIR this is done with the `identifier` field or array type with each item in the array being an object of Identifier type. I don't think we need to have all the FHIR fields but having something basic like

```
"identifier": [{ "value": "2324343", "type": "prime" }, { "value": "222331111", "type": "ssn" }]
```

would be super helpful for extensibility later and supporting the client's SOR ID from the beginning.

Reply:
For reference, the FHIR patient resource looks like this: http://hl7.org/fhir/R4/patient.html

Reply:
also the US Code for patient is this: https://www.hl7.org/fhir/us/core/StructureDefinition-us-core-patient.html

So I think we should at minimum support the core elements (even if we choose not to name them the same or support all fields of nested objects, as in my identifier example above). I think the only one we're missing here is communication language.

Reply:
Yeah I don't think patientID is in any spec list, I just added it because we needed to have some kind of record. I'm on board with an external identifier array!